CONTRIBUTING.md

```diff
@@ -1,7 +1,71 @@
+# Rationale
```

Comment:
It does not make a lot of sense to me to use "Rationale" for the heading here.
Maybe something like "Contributions are welcome" or "Thank you for considering to contribute" could work better as a title?